01-filedir.md

```diff
@@ -426,7 +426,7 @@ and we will see it in many other tools as we go on.
 > 3.  `2012-12-01/ 2013-01-08/ 2013-01-27/`
 > 4.  `original pnas_final pnas_sub`
 
-> ## FIXME {.challenge}
+> ## `ls` command construction {.challenge}
```

Comment:
Great, please go ahead and update your branch @wking. I'll merge when you're done.